dvc/info.py

```diff
@@ -63,6 +66,26 @@ def get_dvc_info():
     return "\n".join(info)
 
 
+def _get_caches(cache):
```

Comment:
Note that we've recently introduced `repo.cache.by_scheme` helper, that should be handy here, as you won't need to parse config SCHEMA anymore.

Reply:
Thanks for sharing, it simplified the logic.